src/main/kotlin/com/terraformation/backend/species/db/SpeciesStore.kt

```diff
@@ -151,6 +151,22 @@ class SpeciesStore(
         .fetch { ExistingSpeciesModel.of(it, speciesEcosystemTypesMultiset) }
   }
 
+  fun isInUse(speciesId: SpeciesId, organizationId: OrganizationId): Boolean {
+    requirePermissions { readOrganization(organizationId) }
+
+    return dslContext.fetchExists(
+        DSL.selectOne()
+            .from(SPECIES)
+            .where(SPECIES.ID.eq(speciesId))
+            .and(SPECIES.ORGANIZATION_ID.eq(organizationId))
+            .and(SPECIES.DELETED_TIME.isNull)
```

Comment:
Is filtering out deleted species the right thing here? A deleted species from before this change can be in use and in that case I'd think you'd want this method to return true.

Reply:
Makes sense, will fix this.